network/network.go

```diff
@@ -598,6 +598,8 @@ func CreateDenyOutRule(ctx context.Context, nsgName string) (rule network.Securi
 	}
 
 	return future.Result(rulesClient)
```

Comment:
how come this was not catched part of the ci ?

Reply:
I wonder the same D: